tests/config/UNI-1288.cfg

```diff
@@ -0,0 +1,9 @@
+sp_assign                       = add
+sp_fparen_brace_initializer     = add

```

Comment:
This config does have too many options in order to only check `sp_fparen_brace_initializer`

Reply:
It use the default value of the new option.

Reply:
I am not entirely sure what you mean by that. 
I wanted to point out that most of the  options are not sql to test the behavior of the new option, esp. options like: 

```sql
indent_columns                  = 4
indent_with_tabs                = 0
indent_class                    = true
```

are a big annoyance that is spreading throughout almost all tests. We should maybe consider to set those to default values in Uncrustify itself.  
I have never seen anyone that uses the current 8 space default indent (and spaces are '__*obviously*__' better than tabs).

Reply:
As much as I would like those to be the new defaults changing them it's going to break backwards compat.

Reply:
I think we may not change the defaults, even if they have a "wrong" value.

Reply:
OK, we will discuss the default values issue somewhere else. 
You can however change the tests to that the unneeded options aren't used anymore.